core/src/main/scala/org/apache/spark/shuffle/IndexShuffleBlockResolver.scala

```diff
@@ -196,12 +196,14 @@ private[spark] class IndexShuffleBlockResolver(
   override def getBlockData(blockId: ShuffleBlockId): ManagedBuffer = {
     // The block is actually going to be a range of a single map output file for this map, so
     // find out the consolidated file, then the offset within that from our index
+    logDebug(s"Fetch block data for $blockId")
```

Comment:
Not necessary to add this, I guess this is mainly for your debug purpose.

Reply:
Ok, I will remove it.

Reply:
Without this info, it looks hard to know continuous shuffle block read really happen, and I found `getLocalBytes` had similar debug info also.
```
logDebug(s"Getting local block $blockId as bytes")
```
How about keeping it?